inc/fields/class-field-post-select.php

```diff
@@ -137,15 +137,15 @@ public function action_wp_ajax_shortcode_ui_post_field() {
 		$query_args['perm']   = 'readable';
 
 		if ( isset( $_GET['page'] ) ) {
-			$query_args['paged'] = sanitize_text_field( $_GET['page'] );
+			$query_args['paged'] = sanitize_text_field( $_GET['page'] ); // WPCS: input var okay
 		}
 
 		if ( ! empty( $_GET['s'] ) ) {
-			$query_args['s'] = sanitize_text_field( $_GET['s'] );
+			$query_args['s'] = sanitize_text_field( $_GET['s'] ); // WPCS: input var okay
 		}
 
 		if ( ! empty( $_GET['post__in'] ) ) {
-			$post__in = is_array( $_GET['post__in'] ) ? $_GET['post__in'] : explode( ',', $_GET['post__in'] );
+			$post__in = is_array( $_GET['post__in'] ) ? $_GET['post__in'] : explode( ',', $_GET['post__in'] ); // WPCS: input var okay
```

Comment:
This works for me:
```php

if ( ! empty( $_GET['post__in'] ) ) {
	$post__in = is_array( sanitize_text_field( $_GET['post__in'] ) ) ? array_map( 'intval', $_GET['post__in'] ) : explode( ',', sanitize_text_field( $_GET['post__in'] ) );
}
```

Handling something that can be an array or a string is just going to be ugly.